scrapers/HelixStudios.yml

```diff
@@ -2,16 +2,47 @@ name: "HelixStudios"
 sceneByURL:
   - action: scrapeXPath
     url:
-      - https://www.helixstudios.com/video/
+      - helixstudios.com/video/
     scraper: sceneScraper
 
+sceneByName:
+  action: scrapeXPath
+  queryURL: https://www.helixstudios.com/videos?q={}
+  scraper: sceneSearch
+
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+
 performerByURL:
   - action: scrapeXPath
     url:
-      - https://www.helixstudios.com/model/
+      - helixstudios.com/model/
     scraper: performerScraper
 
+performerByName:
+  action: scrapeXPath
+  queryURL: https://www.helixstudios.com/models/?q={}
+  scraper: performerSearch
+
 xPathScrapers:
+  sceneSearch:
+    common:
+      $searchItem: //div[contains(@class, "browse-results-grid thumbnail-grid pure-g")]
+      $searchThumb: //div[contains(@class, "browse-results-grid thumbnail-grid pure-g")]//div[contains(@class, "grid-item-wrapper")]/a
+
+    scene:
+      Title: $searchThumb//h4/text()
+      URL:
+        selector: $searchItem//div[contains(@class, "grid-item-wrapper")]/a/@href
+        postProcess:
+          - replace:
+            - regex: ^
+              with: "https://helixstudios.com"
+      Image:
+        selector: $searchItem//div[contains(@class, "image-wrapper img-poster")]//img/@src
```

Comment:
in some cases eg search for `bareback` the image `@src` returns this `data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==` instead of url
switching to `@data-src` seems to work better

Reply:
Upon testing this, @data-src is definitely more reliable, so I've pushed another commit to incorporate this.

Thanks for the suggestion.